app/api-handlers/multiple-error-helper.js

```diff
@@ -126,8 +146,8 @@ module.exports = {
             if (lastTableItem === null || lastTableItem.errorCode !== item.errorCode) {
                 // Create a new display item for the current error code
                 tableItem = {
-                    "fieldName": item.fieldName || "Multiple",
-                    "fieldHeadingText": item.fieldName || "multiple",
+                    "fieldName": errorDataHelper(item.errorData).fieldName,
+                    "fieldHeadingText": errorDataHelper(item.errorData).fieldName,
```

Comment:
fieldHeadingText should have a lowercase version of the term "multiple" as it is used in sentences - your change breaks this.  Perhaps return a third property from errorDataHelper?